src/uncrustify.cpp

```diff
@@ -1583,9 +1583,9 @@ static void do_source_file(const char *filename_in,
 
    if (cpd.if_changed)
    {
-      for (deque<UINT8>::const_iterator i = cpd.bout->begin(), end = cpd.bout->end(); i != end; ++i)
+      for (unsigned char i : *cpd.bout)
```

Comment:
This should be UINT8 and not 'unsigned char'. Yes, they are likely to be the same underlining type, but it is cleaner to use the same type of the container.

Reply:
Good point. Thanks.